src/app/upgrade.cpp

```diff
@@ -110,13 +111,121 @@ namespace
             settingsStorage->removeValue(oldKey);
         }
     }
+
+    void upgradeSchedulerDaysSettings()
+    {
+        auto *settingsStorage = SettingsStorage::instance();
+        const auto key = QString::fromLatin1("Preferences/Scheduler/days");
+        const auto value = settingsStorage->loadValue<QString>(key);
+
+        bool ok = false;
+        const auto number = value.toInt(&ok);
+
+        if (ok)
+        {
+            switch (number)
+            {
+            case 0:
+                settingsStorage->storeValue(key, QLatin1String("EveryDay"));
```

Comment:
You should use `Scheduler::Days` instead of strings here.
The same is about other conversion below.

Reply:
I think not, the migration will break if someone modified the `enum class`.